css/templates/lune/components/_header.css

```diff
@@ -0,0 +1,51 @@
+@import "../page-vars";
+
+.lune .ampstart-headerbar {
+  display: flex;
+  align-items: center;
+
+  height: var(--layout-header-height-m);
+
+  box-shadow: none;
+
+  will-change: transform;
+}
+
+.lune .ampstart-headerbar-title {
+  position: absolute;
+  top: 50%;
```

Comment:
what are we trying to do here , this is something that would work by default on the headerbar (if you are trying to center this)